openlibrary/catalog/edition_merge/templates/index.html

```diff
@@ -1,61 +1,69 @@
 <html>
+
 <head>
-<title>Merge duplicate editions</title>
-<style>
-td { vertical-align: top; background: #ddd };
-</style>
+  <title>Merge duplicate editions</title>
+  <style>
+    td {
```

Comment:
Why is this needed if its also in the legacy stylesheet?

Reply:
DONE